charts/celestia-node/values.yaml

```diff
@@ -19,7 +20,7 @@ config:
 
 images:
   pullPolicy: IfNotPresent
-  node: ghcr.io/celestiaorg/celestia-node:v0.16.0-rc0
+  node: ghcr.io/celestiaorg/celestia-node:v0.18.2-mocha
```

Comment:
Do any of these settings only work on mocha? Noting that we will have use a different image for mainnet instances.